actors/sunbird-lms-mw/actors/common/src/test/java/org/sunbird/learner/actors/coursemanagement/CourseBatchManagementActorTest.java

```diff
@@ -152,18 +170,28 @@ private Response getMockCassandraRecordByIdResponse(int batchProgressStatus) {
     if (batchProgressStatus == ProjectUtil.ProgressStatus.STARTED.getValue()) {
 
       existingStartDate = calculateDate(-4);
-      existingEndDate = calculateDate(2);
+      existingEnrollmentEndDate = calculateDate(1);
+      existingEndDate = calculateDate(3);
+      courseResponseMap.put(JsonKey.COUNTER_INCREMENT_STATUS, Boolean.TRUE);
+      courseResponseMap.put(JsonKey.COUNTER_DECREMENT_STATUS, Boolean.FALSE);
     } else if (batchProgressStatus == ProjectUtil.ProgressStatus.NOT_STARTED.getValue()) {
 
       existingStartDate = calculateDate(2);
+      existingEnrollmentEndDate = calculateDate(3);
       existingEndDate = calculateDate(4);
+      courseResponseMap.put(JsonKey.COUNTER_INCREMENT_STATUS, Boolean.FALSE);
+      courseResponseMap.put(JsonKey.COUNTER_DECREMENT_STATUS, Boolean.TRUE);
     } else {
 
       existingStartDate = calculateDate(-4);
+      existingEnrollmentEndDate = calculateDate(-3);
       existingEndDate = calculateDate(-2);
+      courseResponseMap.put(JsonKey.COUNTER_INCREMENT_STATUS, Boolean.FALSE);
+      courseResponseMap.put(JsonKey.COUNTER_DECREMENT_STATUS, Boolean.FALSE);
```

Comment:
when completed both COUNTER_INCREMENT_STATUS and COUNTER_DECREMENT_STATUS should be true